src/main/java/edu/greenblitz/robotName/subsystems/arm/roller/Roller.java

```diff
@@ -41,6 +41,7 @@ public void periodic() {
 
         roller.updateInputs(rollerInputs);
         Logger.processInputs("Roller", rollerInputs);
+        Logger.recordOutput("Roller/IsObjectIn", isObjectIn());
```

Comment:
Can we prettify it? Made log it in inputs?

Reply:
No because the update should happen in roller and not in neo roller